src/App.tsx

```diff
@@ -17,18 +66,19 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter filter={filter} setFilter={setFilter} searchQuery={searchQuery} setSearchQuery={setSearchQuery}/>
```

Comment:
Ensure that the `TodoFilter` component correctly handles the `filter`, `setFilter`, `searchQuery`, and `setSearchQuery` props. Verify that these props are implemented in the `TodoFilter` component.